Makefile

```
@@ -1,15 +1,19 @@
 CHART_DIR	:= helm-charts
 GIT_VERSION	?= $(shell git describe --dirty=-unsupported --always --long --tags)
+HELM_IMAGE	?= infoblox/helm:3.2.4-5b243a2
+DOCKER_FLAGS	?= --entrypoint="" \
+                        --network host \
+                        -e KUBECONFIG=/apps/.kube/$(notdir $(KUBECONFIG)) \
+                        -v $(dir $(KUBECONFIG)):/apps/.kube/ \
+                        -v $(PWD):/apps
```

You could more in the `DOCKER_FLAGS`...
```suggestion
DOCKER_RUNNER	?= docker run --rm -i \
                        --entrypoint="" \
                        --network host \
                        -e KUBECONFIG=/apps/.kube/$(notdir $(KUBECONFIG)) \
                        -v $(dir $(KUBECONFIG)):/apps/.kube/ \
                        -v $(PWD):/apps
                        $(HELM_IMAGE)
```